css/style.css

```diff
@@ -112,7 +112,7 @@ custom {
 }
 
 #colour-cell-green :hover {
-    background-color: #00ff00;
+    background-color: lime;
```

Comment:
Any future tests that look for the text `green`  as an interpolated CSS value will instead have to look for `lime` once `green` is passed to the test function.